cli/flags.rs

```diff
@@ -1090,9 +1090,10 @@ fn test_subcommand<'a, 'b>() -> App<'a, 'b> {
         .long("coverage")
         .require_equals(true)
         .takes_value(true)
+        .value_name("coverage_dir")
         .conflicts_with("inspect")
         .conflicts_with("inspect-brk")
-        .help("UNSTABLE: Collect coverage profile data"),
+        .help("Collect coverage profile data into coverage_dir"),
```

Comment:
I have no intention of marking this as stable at this time.

```suggestion
        .help("UNSTABLE: Collect coverage profile data into DIR"),
```

Reply:
Ah right, OK.  I am perhaps confused then as the unstable flag is not required to run the coverage.  I'll add this back.